extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -455,11 +458,12 @@ export class AmpAnalytics extends AMP.BaseElement {
       const argList = match[2] || '';
       let raw = (event && event['vars'] && event['vars'][name]) ||
           (trigger['vars'] && trigger['vars'][name]) ||
-          (this.config_['vars'] && this.config_['vars'][name]);
+          (this.config_['vars'] && this.config_['vars'][name]) ||
+          '';
       if (typeof raw == 'string') {
         raw = this.expandTemplate_(raw, trigger, event, opt_iterations - 1);
```

Comment:
none. except that it makes the flow a little simpler to understand. I can revert to following if it is preferable:

```javascript
        let raw = (event && event['vars'] && event['vars'][name]) ||
            (trigger['vars'] && trigger['vars'][name]) ||
            (this.config_['vars'] && this.config_['vars'][name]);

        if (typeof raw == 'string') {
          raw = this.expandTemplate_(raw, trigger, event, opt_iterations - 1);
        }
        const val = opt_encode
          ? this.encodeVars_(raw != null ? raw : '', name)
          : raw != null ? raw : '';
        return val + argList;
```